### DISCUSSION TITLE
[Projects Beta] Delete close project

### DISCUSSION THREAD

**Discussion by noWayHub** [community/community#11621] [Projects and Issues] [answered]
Hello,

how i can delete the closed project beta in projects in Edit i can remove the project,
but in project beta i can only closed the project but i can`t delete it ? Any advice for that?

**Comment by ot-net** [comment]
I have the same issue and looking for a solution.

**Comment by rn4n** [comment]
https://github.com/github/feedback/discussions/6813#discussioncomment-1545558

**Comment by 2percentsilk** [comment, marked as answer]
Thanks for your patience and feedback! We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes 😄
